cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AuthenticationUtils.java

```diff
@@ -121,26 +121,57 @@ private static void checkValueGreaterOrEqualZero(ArrayList<String> errors, Strin
 
     /**
      * Creates the Volumes used for authentication of Kafka client based components
+     *
      * @param authentication    Authentication object from CRD
      * @param volumeList    List where the volumes will be added
      * @param oauthVolumeNamePrefix Prefix used for OAuth volumes
      * @param isOpenShift   Indicates whether we run on OpenShift or not
+     * @param volumeNamePrefix Prefix used for volume names
+     * @param createOAuthSecretVolumes   Indicates whether OAuth secret volumes will be added to the list
      */
-    public static void configureClientAuthenticationVolumes(KafkaClientAuthentication authentication, List<Volume> volumeList, String oauthVolumeNamePrefix, boolean isOpenShift)   {
-        configureClientAuthenticationVolumes(authentication, volumeList, oauthVolumeNamePrefix, isOpenShift, "", false);
+    public static void configurePKCSClientAuthenticationVolumes(KafkaClientAuthentication authentication, List<Volume> volumeList, String oauthVolumeNamePrefix, boolean isOpenShift, String volumeNamePrefix, boolean createOAuthSecretVolumes)   {
```

Comment:
`PKCS` is a broader group of standards, which one are you referring to within this name? Maybe the PKCS12 as referring to the p12 store file? But aren't we moving away from it with this PR? I am confused about the method name then or there is something I am missing here :-(

Reply:
I will change to PKCS12 as suggested. We are keeping this method, as it's still used by other resources such as Bridge and MirrorMaker as they haven't yet migrated to using PEM format certs.

Reply:
Correct. So yeah just changing to PKCS12 is enough. Thanks!